langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/InternalAzureOpenAiHelper.java

```diff
@@ -280,7 +282,9 @@ public static AiMessage aiMessageFrom(ChatResponseMessage chatResponseMessage) {
                                     .build())
                     .collect(toList());
 
-            return aiMessage(toolExecutionRequests);
+            return isNullOrEmpty(text) ?
```

Comment:
```suggestion
            return isNullOrBlank(text) ?
```

Because later on we check `isNullOrBlank`:
```java
    public AiMessage(String text, List<ToolExecutionRequest> toolExecutionRequests) {
        this.text = ensureNotBlank(text, "text");
        this.toolExecutionRequests = ensureNotEmpty(toolExecutionRequests, "toolExecutionRequests");
    }
```